en/docs/learn/configuring-email-otp.md

```diff
@@ -53,8 +53,14 @@ SendGrid APIs. Follow the instructions in **one** of **Option1** or
         !!! Note
             If you are using Gmail account you have to on "Allow less secure
             apps" in your account respective to the above email sending configurations.
-    
-    3.  Add the following email template to the
+            
+    3. Add following property to `deployment.toml` file in the `IS_HOME/repository/conf` folder to comment out the <module ref="addressing"/> property inorder to avoid syntax errors.
+    ```
+    [disable]
```

Comment:
Let's change the template to make this configuration more aligned with the existing toml configurations.